recipes/sagemaker-studio-dataengineering-extensions/post-link.sh

```diff
@@ -0,0 +1,11 @@
+echo $PREFIX
+PYTHON_PKG_NAME=sagemaker_studio_dataengineering_extensions
+# GET Python version in 3.11 or 3.12 format
+PYTHON_VERSION=$(python -V | cut -d ' ' -f2 | cut -d '.' -f1,2)
+
+cd $PREFIX/lib/python$PYTHON_VERSION/site-packages/$PYTHON_PKG_NAME/sagemaker_spark_monitor_widget && pip install .
+cd $PREFIX/lib/python$PYTHON_VERSION/site-packages/$PYTHON_PKG_NAME/sagemaker_jupyter_server_extension && pip install .
+cd $PREFIX/lib/python$PYTHON_VERSION/site-packages/$PYTHON_PKG_NAME/sagemaker_data_explorer && pip install .
+cd $PREFIX/lib/python$PYTHON_VERSION/site-packages/$PYTHON_PKG_NAME/sagemaker_connection_magics_jlextension && pip install .
+cd $PREFIX/lib/python$PYTHON_VERSION/site-packages/$PYTHON_PKG_NAME/sagemaker_ui_doc_manager_jl_plugin && pip install .
+cd $PREFIX/lib/python$PYTHON_VERSION/site-packages/$PYTHON_PKG_NAME/sagemaker_studio_theme && pip install .
```

Comment:
Someone from core might have to decide if this is acceptable.

Mixing `pip` and `conda` packages is often a recipe for corrupting ones environment.

Reply:
can you help to ping someone from core team?

This is some sub packages with different functionality and we want it to ship in one conda package.

Reply:
The usual way would be to package each as `conda` packages and add them as dependencies.

Reply:
we don't want package each, in the long run we will have them merged as one package. This post-link is just a temp workaround and will be removed in the future.

@conda-forge/core, can someone from core team help to review this?

Reply:
The problem is that `pip` installing into a `conda` environment can break that environment if `pip` decides it wants to clobber any `conda` installed files.

If you really do want to install 7 packages as one, you'd probably be better off downloading the source for each [in their own folder](https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#destination-path) and then run ` {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv` in each of the source folders.

Reply:
yeah this is not ok to do in a post-link step. all this kind of installation should happen during install.

Reply:
All these extensions seem like they should have already been installed.

Reply:
If you want to just install 7 packages why not just do this at build time instead of at link time?

Reply:
Thanks for your responses. 

These suggestions are super helpful!! I realized they are already installed and I don't need a post-link script to install them again. I'll have a revision to remove this script soon.